### PR TITLE
Portfolio: accessibility

### PR DESCRIPTION
**Pull request by bushmusi**
# In this Activity we have done below activity

1. **Page titles** : _No issue found_
2. **Image text alternatives**: _No issue found_
3. **Text headings**: _No issue found_
4. **Color contrast**: _No issue found_
5. **Resize**: We update the container size of the paragraph container for the mobile version [here](https://github.com/bushmusi/Microverse-Portfolio/blob/d2e9b8bb6d533a4839d6fc981e2a825f285dcee8/css/style.css#L581-L586)
6. **Interaction**: _No issue found_
5. **Moving content**: Animation was only for desktop version now we also add for mobile version [here](https://github.com/bushmusi/Microverse-Portfolio/blob/d2e9b8bb6d533a4839d6fc981e2a825f285dcee8/css/style.css#L554-L557)
6. **Multimedia**: _No issue found_
7. **The basic structure of the page**:_No issue found_

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,16 +11,17 @@
     <link rel="stylesheet" href="./css/style.css">
     <title>Portfolio</title>
```
Comment (t-yanick) [inline index.html]: - [ ] Please you page title does not meet accessibility requirements, the page title is not descriptive, please see the screenshot below and kindly fix that

accessibility

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @bushmusi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

 ### I will leave the comment here, since it concerns PR summary###

- [x] Please update your `PR` summary to include a [link to the corresponding line of code in your pull request changes.](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148). Kindly link to the lines of code that you have made changes on. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (bushmusi) [reply index.html]: You are right. I fix that 👋

**Review by medaminedev66 — APPROVED** [review]
Hi @bushmusi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**